_docker/drupal/drupal-filesystem/web/themes/custom/rhd/css/base/eclipse.css

```diff
@@ -0,0 +1 @@
+.wide.wide-hero.eclipse{background:url(/images/eclipse/eclipse_microsite_hero.jpg);background-size:cover;height:450px}
```

Comment:
I have no idea what generates this. I haven't looked into it either. I'd recommend removing it from the PR. If you'd like to take the extra time to figure out what's generating it, even better. First thoughts, it's something in the Gulp script.